### PR TITLE
fix(validator): Allow form data will mutliple values appended

### PR DESCRIPTION
**Pull request by nicksrandall**
### The author should do the following, if applicable

- [x] Add tests
- [x] Run tests
- [x] `bun run format:fix && bun run lint:fix` to format the code
- [x] Add [TSDoc](https://tsdoc.org/)/[JSDoc](https://jsdoc.app/about-getting-started) to document the code

### CODE REVIEW

File: src/validator/validator.ts [hunk 1]
```diff
@@ -108,11 +108,11 @@ export const validator = <
...
         formData.forEach((value, key) => {
           if (key.endsWith('[]')) {
-            if (form[key] === undefined) {
-              form[key] = [value]
-            } else if (Array.isArray(form[key])) {
-              ;(form[key] as unknown[]).push(value)
-            }
+            ;((form[key] ??= []) as unknown[]).push(value)
+          } else if (Array.isArray(form[key])) {
+            ;(form[key] as unknown[]).push(value)
```
Comment (yusukebe) [inline src/validator/validator.ts:113]: @nicksrandall Can you write the test to cover this line?

Comment (yusukebe) [reply src/validator/validator.ts:113]: Okay!

Comment (nicksrandall) [reply src/validator/validator.ts:113]: I *think* it might be better but the coverage action is failing in CI so I'm not sure

Comment (yusukebe) [reply src/validator/validator.ts:113]: @nicksrandall 

The CI seemed to fail because of accidentally. Restared, now it passed.

**Review by yusukebe — APPROVED** [review]
LGTM!

**Comment by yusukebe** [comment]
@nicksrandall 

Thanks! It's a good fix. I'll merge this now.

**Pull request merged**